systers_portal/templates/community/snippets/search-snippet.html

```diff
@@ -1,3 +1,26 @@
+<script>
+
+
+  for (var i = 0; i < markers.length; i++) {
+            markers[i].setMap(null);
+            console.log("deleting",marker[i])
+          }
+           var markers = [];
+        {% for community in communities %}
+          var point = new google.maps.LatLng({{community.location.latitude}},{{community.location.longitude}});
+          var marker = new google.maps.Marker({position: point, map: map, title: '{{ community.name }}',});
+          marker['infowindow'] = new google.maps.InfoWindow({content: 
+            "<a href='{% url "view_community_landing" community.slug %}'>{{community}}</a>", });
+          google.maps.event.addListener(marker, 'click', function() {this['infowindow'].open(map, this);});
+          markers.push(marker);
+            
+      {% endfor %} 
+      console.log(markers.length)
```

Comment:
remove these if you are done with them?

Reply:
Oh I'm sorry.. left this by mistake.  Including just templates with JavaScript code can be done i think. i will try that out.